operatorhub/openshift/manifests/bases/openshift-pipelines-operator-rh.clusterserviceversion.template.yaml.tmpl

```diff
@@ -315,6 +315,7 @@ spec:
     - Tekton Hub (tech-preview): {{ with index .Versions "hub"}}{{ .Version }}{{end}}
     - Tekton Results (tech-preview): {{ with index .Versions "results"}}{{ .Version }}{{end}}
     - Manual Approval Gate (tech-preview): {{ with index .Versions "manual-approval-gate"}}{{ .Version }}{{end}}
+    - Tekton Pruner (tech-preview): {{ with index .Versions "pruner"}}{{ .Version }}{{end}}
```

Comment:
You need to add the defination of Pruner in CSV similar to how other CRs are added
ex: [manual-approval-gate](https://github.com/tektoncd/operator/blob/main/operatorhub/openshift/manifests/bases/openshift-pipelines-operator-rh.clusterserviceversion.template.yaml.tmpl#L218)